docs/utilities/data_masking.md

```diff
@@ -425,15 +426,35 @@ You can modify the following values when initializing the `AWSEncryptionSDKProvi
 | **max_messages_encrypted** | `4294967296`          | The maximum number of messages that may be encrypted under a cache entry                      |
 | **max_bytes_encrypted**    | `9223372036854775807` | The maximum number of bytes that may be encrypted under a cache entry                         |
 
-**Changing the default algorithm**
+If required, you have the option to customize the default values when initializing the `AWSEncryptionSDKProvider` class.
```

Comment:
```suggestion
If required, you can customize the default values when initializing the `AWSEncryptionSDKProvider` class.
```

`You have the option to` becomes `You can`

You could go even further in the future when shorter length is necessary by using synonyms, for example.

`You have the option to customize` becomes `You can change`